src/wp-includes/blocks/index.php

```diff
@@ -12,6 +12,61 @@
 require BLOCKS_PATH . 'widget-group.php';
 require BLOCKS_PATH . 'require-dynamic-blocks.php';
 
+/**
+ * Registers core block style handles.
+ *
+ * While {@see register_block_style_handle()} is typically used for that, the way it is
+ * implemented is inefficient for core block styles. Registering those style handles here
+ * avoids unnecessary logic and filesystem lookups in the other function.
+ *
+ * @since 6.3.0
+ */
+function register_core_block_style_handles() {
+	if ( ! wp_should_load_separate_core_block_assets() ) {
+		return;
+	}
+
+	static $core_blocks_meta;
+	if ( ! $core_blocks_meta ) {
+		$core_blocks_meta = require ABSPATH . WPINC . '/blocks/blocks-json.php';
+	}
+
+	$suffix = wp_scripts_get_suffix();
+	$wp_styles = wp_styles();
+	$style_fields = array(
+		'editorStyle',
+		'style',
+	);
+
+	foreach ( $core_blocks_meta as $name => $schema ) {
+		foreach ( $style_fields as $style_field ) {
+			if ( ! isset( $schema[ $style_field ] ) ) {
+				$style_handle = 'style' === $style_field ? "wp-block-{$name}" : "wp-block-{$name}-editor";
+				$wp_styles->add(
+					$style_handle,
+					false
+				);
+				continue;
+			}
+			$style_handle = $schema[ $style_field ];
+			if ( is_array( $style_handle ) ) {
+				continue;
+			}
+			$path = "/wp-includes/blocks/{$name}/style{$suffix}.css";
+			$wp_styles->add( $style_handle, $path );
+			$wp_styles->add_data( $style_handle, 'path', ABSPATH . $path );
```

Comment:
```suggestion
			$path = WPINC . "/blocks/{$name}/style{$suffix}.css";
			$wp_styles->add( $style_handle, "/{$path}" );
			$wp_styles->add_data( $style_handle, 'path', ABSPATH . $path );
```

`ABSPATH . $path` added duplicate forward slash. Let's follow similar approach that used here https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/script-loader.php#L1600-L1602

**Image:**
<img width="938" alt="Screenshot 2023-06-20 at 10 18 50 AM" src="https://github.com/WordPress/wordpress-develop/assets/10103365/42b8ce7f-8c65-4c5d-acfc-4915a3fa6cb3">